Source/GSXib5KeyedUnarchiver.m

```diff
@@ -381,7 +382,8 @@ + (void) initialize
                @"decodeDividerStyleForElement:", @"NSDividerStyle",
                @"decodeToolbarIdentifiedItemsForElement:", @"NSToolbarIBIdentifiedItems",
                @"decodeToolbarImageForElement:", @"NSToolbarItemImage",
-               nil];
+               @"decodeControlContentsForElement:",@"NSControlContents",
```

Comment:
Where does this key get used? I could not find it anywhere in the GNUstep gui code.

Reply:
I don't think it does.  I will remove it.  It was added earlier...  I was trying to understand what was going on in GSXib5KeyedUnarchiver.

Reply:
It is used in NSSwitch.   It decodes the "switch='on'" attribute.

Reply:
As I wrote I do not use 10.15 and won't be able to see that class in action myself. Could you please provide me with a key encoded file for NSSwitch? It would be best in XML format. 
The code I use for that looks something like this:

void encode(id object)
{
  NSMutableData *m;
  NSKeyedArchiver *a;
  NSString *aPath;
  
  aPath = @"/Users/fred/Documents/GNUstep/keyedcoding/keyed.xml";

  m = [[NSMutableData alloc] initWithCapacity: 10240];
  a = [[NSKeyedArchiver alloc] initForWritingWithMutableData: m];
  [a setOutputFormat: NSPropertyListXMLFormat_v1_0];
  [a encodeObject: object forKey: @"root"];
  [a finishEncoding];
  [m writeToFile: aPath atomically: YES];
  [a release];
  [m release];
}

Reply:
Also a space after the comma is missing.

Reply:
I am trying to attach it here, but it's not cooperating.   I will email them to you.

Reply:
I sent them.